services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/service/implement/SubnetServiceImp.java

```diff
@@ -487,7 +487,7 @@ public void updateSubnetRoutingRuleInRM(String projectId, String subnetId, Subne
         routetable.setRouteTableType(RouteTableType.NEUTRON_SUBNET.getRouteTableType());
         routetable.setRouteEntities(routeEntities);
 
-        String routeManagerServiceUrl = routeUrl + "project/" + projectId + "/subnets/" + subnetId + "/routetable";
+        String routeManagerServiceUrl = routeUrl + "project/" + projectId + "/subnets/" + subnetId + "/routetable?routerid=" + subnetEntity.getAttachedRouterId();
```

Comment:
Do we always preassign value to SubnetEntity's attachedRouterId?

Reply:
Yes.

Reply:
Yes, this is for host route.